docs/user-guide/configure-load-balancing.md

```diff
@@ -2,6 +2,7 @@
 
 Styx supports three load balancing strategies:
 
+ - Power of Two
  - Round robin
```

Comment:
I think this is usually spelt 'round-robin', with a hyphen. In that case we should probably change all the later occurrences (like in the header).